### ISSUE TITLE
Maven build errors

### ISSUE THREAD

**Issue by lhstrh** [lf-lang/lingua-franca#791]
The Maven build, albeit successful, spews a lot of error messages like the following:

```
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/ui/ColorHexUtil.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/ui/PlaceHolder.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/ui/DummyIconManager$DummyRowIcon.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/ui/IconManager.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/ui/IconManagerHelper.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/ui/DummyIconManager.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/ui/PlaceProvider.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/ui/DummyIconManager$1.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/ui/DummyIconManager$DummyIcon.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/psi/codeStyle/ChangedRangesInfo.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/psi/codeStyle/DocCommentSettings$Defaults.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/psi/codeStyle/SuggestedNameInfo$1.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/psi/codeStyle/SuggestedNameInfo$Delegate.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/psi/codeStyle/SuggestedNameInfo.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/psi/codeStyle/FormattingModeAwareIndentAdjuster.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/psi/codeStyle/Indent.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/psi/codeStyle/CodeStyleManager.class
[ERROR] Could not merge ide-dependencies.jar Conflicting file: com/intellij/psi/codeStyle/DocCommentSettings.class
```

**Comment by petervdonovan** [comment]
I think that I am to blame for that; the VS Code branch has had this issue for some time. I will take a look.

**Comment by petervdonovan** [comment]
This appears to be closed by #795. Many thanks to @a-sr.
